### DISCUSSION TITLE
How to use nested routing in a code based manner

### DISCUSSION THREAD

**Discussion by zeroanonx** [TanStack/router#2834] [Q&A]
> [@LinHanlove](https://github.com/LinHanlove) not sure what you mean. can you please create a new issue / discussion including an example to explain your questions ?

Firstly, I used code based routing. I created a 'parent component' - 【 Home 】, which can redirect normally. I also created a child component based on home - 【 homeChild 】, and their paths are as follows

-/home - Jump to the Home page

-/home/child - Jump to HomdChild under Home

However, this is incorrect, it cannot be redirected, Not Found

 follows： 
![Image](https://github.com/user-attachments/assets/0b4ccedb-0ac5-447f-bc38-10428b10f78f)
![Image](https://github.com/user-attachments/assets/06d1884e-8d92-4404-94ce-0c8082afcafd)
![Image](https://github.com/user-attachments/assets/4b7660d4-dd1b-4874-977d-6d757f23e29d)

**Comment by schiller-manuel** [comment]
Since you just provided screenshots, I can only guess.
Looks like you should not provide the full path `/home/child` for the child route path, instead just use `child`

**Reply by zeroanonx** [reply]
no, /child  It's also wrong Please take a look here。https://github.com/TanStack/router/issues/2835
